languages.yaml

```diff
@@ -1,48 +0,0 @@
-en:
-    THEME_LEARN2_GITHUB_EDIT_THIS_PAGE: edit this page
-    THEME_LEARN2_GITHUB_NOTE: Found errors? Think you can improve this documentation?
-    THEME_LEARN2_CLEAR_HISTORY: Clear History
-    THEME_LEARN2_BUILT_WITH_GRAV: Built with <a href="http://getgrav.org">Grav</a> - The Modern Flat File CMS
-    THEME_LEARN2_SEARCH_DOCUMENTATION: Search Documentation
-cn:
```

Comment:
We are losing support for some languages by simply deleting this file and replacing it with en, fr.  You should at least create files for these languages with only these few translations in them.